src/lib/mina/account-update.ts

```diff
@@ -614,6 +614,10 @@ const Body = {
   ): Body {
     let { body } = Types.AccountUpdate.empty();
     body.publicKey = publicKey;
+
+    // useFullCommitment = true by default, prevents replay attacks and similar issues
+    // the developer can still modify this to false if needed later on
+    body.useFullCommitment = Bool(true);
```

Comment:
I'm not sure about this, how about setting this to true in `requireSignature()` instead of always? it's a bit weird to me that the default account update differs from the natural dummy AU with no apparent reason

Reply:
or in `setLazySignature`?

Reply:
well or maybe we should remove `setLazySignature` since it no longer holds a private key so no real purpose